### PR TITLE
Generate Sbom for VS insertions

### PR DESCRIPTION
**Pull request by epananth**
Fixes #

### Context

Generate SBOMs for VS insertions 
### Testing
Will run a test after the arcade update PR is merged.

### CODE REVIEW

**Comment by epananth** [comment]
Need to merge https://github.com/dotnet/msbuild/pull/7808 before I can merge this one

**Comment by Forgind** [comment]
I merged #7808. Let me know when you've tested this one; the change itself looks good to me, though the space change will conflict with #7796 if we end up merging that.

**Comment by epananth** [comment]
> I merged #7808. Let me know when you've tested this one; the change itself looks good to me, though the space change will conflict with #7796 if we end up merging that.

Idk why bootstrapper got added in my commit. Removing it.

**Comment by epananth** [comment]
Will merge this after the new arcade update comes through tomo morning

**Comment by epananth** [comment]
This should be good to merge now.

File: .vsts-dotnet.yml
Comment (Forgind) [inline .vsts-dotnet.yml:151]:
```suggestion
        bootstrapperCoreVersion: 
```

**Review by Forgind — APPROVED** [review]

**Comment by Forgind** [comment]
Thanks for driving this through, @epananth! It'll be nice to not have to manually make the SBoM optional every time 😃

**Pull request merged**
